cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -298,6 +303,19 @@ protected void configure() {
           })
       );
 
+      install(
+        new FactoryModuleBuilder()
+          .implement(Configurator.class, InMemoryConfigurator.class)
+          .build(Key.get(ConfiguratorFactory.class, Names.named("local")))
+      );
+      install(
+        new FactoryModuleBuilder()
+          .implement(Configurator.class, InMemoryConfigurator.class)
```

Comment:
Is this binding correct? Shouldn't it bind to the `RemoteConfigurator`?

Reply:
Whoops, thats not right. Thanks